buddyboss/buddyboss-platform
# Fixed - Delete account button is not working from Account > Delete Ac…

…count page for users

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2055  .

### How to test the changes in this Pull Request:

1. Go to delete account page
2. Checked "I understand the consequences." checkbox
3. Delete Account button should active now

### Proof Screenshots or Video

https://www.loom.com/share/693ee8e4cc284d94a243b4144bc819f3



### Other information:
